common/djangoapps/util/tests/test_submit_feedback.py

```diff
@@ -463,7 +462,8 @@ def test_valid_request_auth_user_with_custom_fields(self, course_id, enrolled, z
         ("course-v1:testOrg+testCourseNumber+testCourseRun", False),
     )
     @unpack
-    @override_settings(ZENDESK_CUSTOM_FIELDS=TEST_ZENDESK_CUSTOM_FIELD_CONFIG, ENABLE_ENTERPRISE_INTEGRATION=True)
+    @override_settings(ZENDESK_CUSTOM_FIELDS=TEST_ZENDESK_CUSTOM_FIELD_CONFIG)
+    @mock.patch.dict("django.conf.settings.FEATURES", dict(ENABLE_ENTERPRISE_INTEGRATION=True))
```

Comment:
Why not use `override_settings` to do this?

Reply:
@douglashall  I found that if I use `override_settings`, then I end up overriding the whole `FEATURES` block, that that messes with the [`@mock.patch` `FEATURES` that wrap the test class](https://github.com/edx/edx-platform/pull/16155/files#diff-8f395611f456cfd137e4252981eedf31L48).